php/Terminus/Commands/SitesCommand.php

```diff
@@ -486,44 +486,40 @@ function ($site) {
         return $filtered_sites;
     }
 
-  /**
-=======
->>>>>>> Fixed sites list memberships so that information is correctly rendered when a user is both an organizational and a team member.
-   * A helper function for getting/prompting for the site create options.
-   *
-   * @param array $assoc_args Arguments from command
-   * @return array
-   */
+    /**
+     * A helper function for getting/prompting for the site create options.
+     *
+     * @param array $assoc_args Arguments from command
+     * @return array
+     */
     private function getSiteCreateOptions($assoc_args)
     {
-        $options          = [];
-        $options['label'] = $this->input()->string(
-            [
-            'args'     => $assoc_args,
-            'key'      => 'label',
-            'message'  => 'Human-readable label for the site',
-            'required' => true,
-            ]
-        );
-        $suggested_name   = $this->sanitizeName($options['label']);
-
+        $options = [
+          'label' => $this->input()->string(
+              [
+              'args'     => $assoc_args,
+              'key'      => 'label',
+              'message'  => 'Human-readable label for the site',
+              'required' => true,
+              ]
+          ),
+        ];
         if (array_key_exists('name', $assoc_args)) {
             // Deprecated but kept for backwards compatibility
             $options['site_name'] = $assoc_args['name'];
         } elseif (array_key_exists('site', $assoc_args)) {
             $options['site_name'] = $assoc_args['site'];
-        } elseif (!is_null(Config::get('site'))) {
-            $options['site_name'] = Config::get('site');
+        } elseif (!empty($site = Config::get('site'))) {
```

Comment:
This line right here was the culprit. The change in config meant that we now have to check for empty rather than null.